### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

File: WeakAuras/Animations.lua [hunk 1]
```diff
@@ -382,7 +382,9 @@ function Private.Animate(namespace, uid, type, anim, region, inverse, onFinished
       last_update = GetTime()
       updatingAnimations = true;
     end
-    RunAnimation(key, animation, 0, GetTime())
+    C_Timer.After(0, function()
+      RunAnimation(key, animation, 0, GetTime())
+    end)
```
Comment (Stanzilla) [inline WeakAuras/Animations.lua:387]: lovely

Comment (InfusOnWoW) [reply WeakAuras/Animations.lua:387]: Yes indeed... No way that'll create problems in the future.

Comment (InfusOnWoW) [reply WeakAuras/Animations.lua:387]: Actually I have an idea...

Comment (InfusOnWoW) [reply WeakAuras/Animations.lua:387]: Nah, I won't tinkering with it for now.

**Review by mrbuds — APPROVED** [review]

**Pull request merged**